Added basic broadcast functionality to app.

jsnprtr opened the pull request:
#### What's this PR do?
Adds basic broadcast functionality to the app.

##### Background context
Making the game multiplayer using EventMachine

#### Where should the reviewer start?
connector.rb - has no tests...not sure how to test :(

#### How should this be manually tested?
Play the game, load a new player in a new browser tab

#### Screenshots
![kapture 2019-03-02 at 10 01 28](https://user-images.githubusercontent.com/2657482/53681216-9a470100-3ce6-11e9-8e59-a8ef8d67079f.gif)

Dannyguk commented on app/app.rb:
```diff
@@ -20,16 +20,14 @@ def run
-        EventMachine::WebSocket.start(host: @socket_host, port: @socket_port) do |ws|
 
+        channel = EM::Channel.new
+        @count = 0;
```
If i understand correctly, i think count might be replaced by player UUID, but we could do that later.

Dannyguk commented on app/lib/connection.rb:
```diff
@@ -0,0 +1,25 @@
+module Hammeroids
```
There's some style weirdness/inconsistency in here with {} spacing around blocks, also not sure i get the calls in the constructor, but could refactor later?

jsnprtr commented:
@Dannyguk I removed the @count from app.rb because it was just getting overwritten by the channel subscribe method. We can change this to UUID later though, I think that would be better
I moved all the socket setup to the constructor in connection.rb and fixed the whitespace around braces.
